### PR TITLE
docs: enhancing release 24.03. system documentation

### PR DESCRIPTION
**Pull request by jjeroch**
## Description

Adding additional content to architecture documentation as well as image 

## Why

Getting ready for release 24.03.

## Issue

https://github.com/eclipse-tractusx/portal-assets/issues/255

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/policy-hub/blob/main/docs/technical-documentation/dev-process/How%20to%20contribute.md)

### CODE REVIEW

**Comment by jjeroch** [comment]
@Phil91  & @evegufy I guess we still need to create a "static" folder for the image.
Can do this later today

**Comment by Phil91** [comment]
@jjeroch I'd wait until #56 is merged, there I did add a mermaid diagram instead of a picture for the Whitebox Overall System to comply with the trgs

**Review by Phil91 — APPROVED** [review]

**Pull request merged**
